src/aspire/reconstruction/mean.py

```diff
@@ -139,7 +139,7 @@ def _compute_kernel(self):
                     if j != k:
                         kernel[j, k] += batch_kernel
 
-        kermat_f = np.zeros((self.r, self.r, _2L, _2L, _2L))
+        kermat_f = np.zeros((self.r, self.r, _2L, _2L, _2L), dtype=self.dtype)
```

Comment:
Note: Found this while switching out `FBBasis3D` for `FFBBasis3D` in the Cov3D tutorial.